rllab/envs/mujoco/point_env.py

```diff
@@ -1,10 +1,10 @@
-from rllab.envs.base import Step
+import glfw
+import math
 from .mujoco_env import MujocoEnv
```

Comment:
PEP8: no relative imports

Reply:
glfw is an absolute import:

/Library/Frameworks/Pyton.framework/Versions/3.6/lib/python3.6/site-packages/glfw/__init__.py

Reply:
```python
from .mujoco_env import MujocoEnv
```